internal/config/config.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+import "time"
+
+type TLS struct {
+	CACertFile string
+	CertFile   string
+	KeyFile    string
+}
+
+type Config struct {
+	Addr            string
+	TLS             TLS
+	ShutdownTimeout time.Duration
+	CPUMax          string
+	MemoryMax       string
+	IOMax           []string
```

Comment:
Should these properties be a string?

Reply:
The challenge requirements said not to spend much time on config, so I chose the simplest thing possible here which was to pass in the literal values that would be used in the cgroup files. Probably not what I would choose in any other case, but it is easy and works.

Reply:
Updated with better typing and validation